miniapps/gslib/findpts.cpp

```diff
@@ -32,7 +32,8 @@
 //    findpts -m ../../data/inline-quad.mesh -o 3
 //    findpts -m ../../data/inline-quad.mesh -o 3 -po 1
 //    findpts -m ../../data/inline-quad.mesh -o 3 -po 1 -fo 1 -nc 2
-//    findpts -m ../../data/inline-quad.mesh -o 3 -hr -pr
```

Comment:
Should the old sample run be kept?

Reply:
Also, would it be difficult to add the new support to `pfindpts.cpp`?

Reply:
Updated the sample run.
Variable order spaces are not yet supported in parallel so only updated the serial miniapp.

Reply:
Should this old run be kept? `findpts -m ../../data/inline-quad.mesh -o 3 -hr -pr`

Reply:
I think the updated run can replace the old run.